test/test/stackwalker/StackwalkerTests.java

```diff
@@ -84,4 +85,47 @@ public void normalStackVM(TestProcess p) throws Exception {
                 "Java_test_stackwalker_StackGenerator_leafFrame;" +
                 "doCpuTask");
     }
+
+    @Test(mainClass = StackGenForUnwindViaDebugFrame.class, jvmArgs = "-Xss5m",
+            agentArgs = "start,event=cpu,cstack=vm,file=%f.jfr", os = Os.LINUX)
+    public void walkStackUsingDebugFrameVM(TestProcess p) throws Exception {
+        p.waitForExit();
+        assert p.exitCode() == 0;
+        Output output = Output.convertJfrToCollapsed(p.getFilePath("%f"));
```

Comment:
Is there a specific reason to use JFR in the first place? I think it adds one layer of complexity to the test